Gender Analysis.ipynb

```diff
@@ -1699,7 +1700,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.3"
+   "version": "3.7.9"
```

Comment:
remove, please don't change python version

Reply:
I'm not sure how to change this, I think it just happened because I ran the notebook with 3.7.9 because that's what's on my OS.

Reply:
you can manually change it in your commit so that you don't push this change